### ISSUE TITLE
Provide events.json

### ISSUE THREAD

**Issue by jchesterpivotal** [jchesterpivotal/concourse-build-resource#20]
## Description

Currently, the resource renders events out to `events.log`. This is useful for folks wanting raw loglines.

It would be nice to have access to the original JSON form (say `events.json`), however. This would allow analysis of timestamps and grouping into the various steps.

## Todos

* [ ] cmd/in
* [ ] pkg/in
* [ ] Add a `show-events` utility task

**Comment by jchesterpivotal** [comment]
There's not much inbuilt support for this in `go-concourse`, which is a bit of a hassle.

**Comment by jchesterpivotal** [comment]
So a few notes.

1. This really _would_ be super helpful to perform more detailed analyses of build behaviour.
2. Concourse 4.2.1 and 3.14.x (the versions I've been referring to in the wild) have very different levels of detail. In a 4.2.1 instance I can see actual markers that tell me when a `get`, `put` or `task` ends. It also looks as though they provide more detailed metadata about the tasks etc.
3. Authentication (#9) rears its head again. This won't be very useful for lots of folks if it requires them to have public job logs.

**Comment by jchesterpivotal** [comment]
Marking blocked on the authentication story.

**Comment by jchesterpivotal** [comment]
Unblocking. This is possible without authentication, just not as useful in cases of private jobs.

**Comment by jchesterpivotal** [comment]
Also, I was wrong in saying that 3.x and 4.x have different levels of detail about the stuff I am interested in. Mea culpa.
